packages/features/eventtypes/components/tabs/advanced/EventAdvancedTab.tsx

```diff
@@ -102,14 +108,283 @@ export type EventAdvancedBaseProps = Pick<EventTypeSetupProps, "eventType" | "te
+            <div className="h-4 w-32 animate-pulse rounded-md bg-gray-200" />
+            <div className="mt-2 h-10 w-full animate-pulse rounded-md bg-gray-200" />
+          </div>
+        </div>
+        <div className="space-y-2">
+          <div className="h-6 w-64 animate-pulse rounded-md bg-gray-200" />
+          <div className="h-10 w-full animate-pulse rounded-md bg-gray-200" />
+          <div className="h-4 w-48 animate-pulse rounded-md bg-gray-200" />
+        </div>
+      </div>
+    );
+  },
+};
+
+const calendarComponents = {
+  CalendarSettingsSkeleton() {
+    return (
+      <div>
+        <destinationCalendarComponents.DestinationCalendarSettingsSkeleton />
+        <SelectedCalendarsSettingsWebWrapperSkeleton />
+      </div>
+    );
+  },
+
+  CalendarSettings({
+    eventType,
+    calendarsQuery,
+    verifiedSecondaryEmails,
+    userEmail,
+    isTeamEventType,
+    isChildrenManagedEventType,
+    customClassNames,
+    eventNameLocked,
+    eventNamePlaceholder,
+    setShowEventNameTip,
+    showToast,
+  }: CalendarSettingsProps) {
+    const formMethods = useFormContext<FormValues>();
+    /**
+     * Only display calendar selector if user has connected calendars AND if it's not
+     * a team event. Since we don't have logic to handle each attendee calendar (for now).
+     */
+
+    const isConnectedCalendarSettingsApplicable = !isTeamEventType || isChildrenManagedEventType;
+    const isConnectedCalendarSettingsLoading = calendarsQuery.isPending;
+    const showConnectedCalendarSettings =
+      !!calendarsQuery.data?.connectedCalendars.length && isConnectedCalendarSettingsApplicable;
+
+    const selectedCalendarSettingsScope = formMethods.getValues("useEventLevelSelectedCalendars")
+      ? SelectedCalendarSettingsScope.EventType
+      : SelectedCalendarSettingsScope.User;
+
+    const destinationCalendar = calendarsQuery.data?.destinationCalendar;
+    if (isConnectedCalendarSettingsLoading && isConnectedCalendarSettingsApplicable) {
+      return <calendarComponents.CalendarSettingsSkeleton />;
+    }
+
+    return (
+      <div>
+        <destinationCalendarComponents.DestinationCalendarSettings
+          verifiedSecondaryEmails={verifiedSecondaryEmails}
+          userEmail={userEmail}
+          isTeamEventType={isTeamEventType}
+          calendarsQuery={calendarsQuery}
+          eventNameLocked={eventNameLocked}
+          eventNamePlaceholder={eventNamePlaceholder}
+          setShowEventNameTip={setShowEventNameTip}
+          showToast={showToast}
+          showConnectedCalendarSettings={showConnectedCalendarSettings}
+          customClassNames={customClassNames}
+        />
+        <div>
+          {isConnectedCalendarSettingsApplicable
+            ? showConnectedCalendarSettings && (
+                <div className="mt-4">
+                  <Suspense fallback={<SelectedCalendarsSettingsWebWrapperSkeleton />}>
+                    <SelectedCalendarsSettingsWebWrapper
```

Comment:
this component will cause the atom to break since it uses trpc

Reply:
looking into it, might just disable this for platform atm

Reply:
Is there a way to conditionally render it for non-atom scenario for now?

I believe we would have to fetch the data at higher level and pass to it to avoid the requirement of tRPC right?

Reply:
usually yes we use the wrappers to handle this behaviour, web wrapper call trpc, platform wrapper call api v2

Reply:
for now we can use the hook useIsPlatform to not render this field for platform, and I will make a follow up pr to make it work on platform